### DISCUSSION TITLE
Flywoo GP11

### DISCUSSION THREAD

**Discussion by Flybyjames** [gopro/labs#666] [Q&A]
Hello my GP11 has lost bluetooth connection> I have verified it is not the device with the Quik app as I have other camera's that connect without issue. I have tried to repair the connection but the camera will scan the code but the Quik app still does not see the camera to pair. Is there a QR code to turn off and on bluetooth? Not sure why it stopped working. The camera itself works just fine, except for not connecting to the app and it seems to take much longer to power off. Is it safe to reload the Gopro labs firmware or would I need to have a Gopro battery and its usb cable? I have enduro battery but not the cable/connector. If that cable is needed does anyone have a link so I may purchase it? Any advice would be greatly appreciated.
Thanks for taking the time to read this.
FlybyJames

**Comment by dnewman-gpsw** [comment]
It is safe to update Labs, but only do that if an update is needed as there is not value in reinstalling the same FW.  Flywoo does ship with very old Labs FW, so updating to 2.20.70 is a good step first.  You don't need an LCD or original battery as Labs is already installed. As you can remove power, this is turning off BLE, so on/off switch is not the issue, but some messed up credentials could be.

try:
!FRESET   <- factory reset, takes 15-20s, will reboot

then:
!PA           <- pair app

**Reply by Flybyjames** [reply]
I tried the QR codes, still will not connect to the Quik app. I also updated to the newest firmware but that also did not help. I used QR codes to setup the preferences but they don't seem to stick after powering off. Wondering if I may have something defective? I just rechecked my bones and it connects just fine to the app. Not sure how to proceed.

**Reply by dnewman-gpsw** [reply]
How do you mean that you can't select preferences via QR codes? What is not sticking? Resolution, frame rate, bitrate, Protune settings, should all be configurable by QR codes.

**Comment by Flybyjames** [comment]
Thanks very much for the reply and suggestions.
________________________________

**Comment by Flybyjames** [comment]
Ok after rechecking my settings are being saved but I still can't connect to the Quik app. I tried on my phone and my tablet with no luck. I have a question, I tried to reflash the newest firmware thinking maybe there was a issue. When reflashing the camera keeps on trying to load the firmware over and over. it never deletes the update file. It will try to reflash until I eject the SD card and delete the Update file, is this normal? Thanks for all your help, I am very new to this action camera stuff and appreciate all your suggestions.
________________________________

**Reply by dnewman-gpsw** [reply]
That is not normal.  Looks on your SD Card for MISC/version.txt 

It should reflect the install firmware version:

{
"info version":"2.0"
,"firmware version":"H22.01.02.20.70"
,"wifi mac":"247**********"
,"camera type":"HERO11 Black"
,"camera serial number":"C34**********"
}

**Comment by Flybyjames** [comment]
Here is the information, is the camera stuck in a loop because it already has the same firmware version loaded? I would have thought it would just overwrite the file. Would reverting to a previous version then reflashing the newest version be of any help? I'm open for any suggestions.
Currently the camera does work but will not connect to the app. I guess worst case scenario, I could use QR codes to operate the camera but wow that wouldn't be ideal.
{
"info version":"2.0"
,"firmware version":"H22.01.02.20.70"
,"wifi mac":"24*********"
,"camera type":"HERO11 Black"
,"camera serial number":"C34*********"
}
Thanks for all your help
________________________________

**Reply by dnewman-gpsw** [reply]
No, downgrading will not help, something is gone wrong with the hardware, as we are seeing multiple failures (pair and updating issues.)   Time to embrace QR codes. I never use the App, only use QR codes, as I find the App too slow.

**Comment by Flybyjames** [comment]
Ok thanks I will use QR codes,  I normally only use a few different settings, guess I could get used to having some codes ready and just scan them. Thanks for all your help.
________________________________

**Reply by dnewman-gpsw** [reply]
Use Boot commands with wake on power. I use audio triggers, so the camera is recording when the rotors are make sufficient noise.  Never have to remember to start and stop captures.

**Comment by Flybyjames** [comment]
Thanks, I will!
